docs/en/seatunnel-engine/telemetry.md

```diff
@@ -0,0 +1,152 @@
+---
+sidebar_position: 13
+---
+
+# Telemetry
+
+Integrating `Metrices` through `Prometheus-exports` can better seamlessly connect to related monitoring platforms such
+as Prometheus and Grafana, improving the ability to monitor and alarm of the SeaTunnel cluster.
+
+You can configure telemetry's configurations in the `seatunnel.yaml` file.
+
+The following is an example declarative configuration.
+
+```yaml
+seatunnel:
+  engine:
+    telemetry:
+      metric:
+        enabled: true # Whether open metrics export
+```
+
+## Metrics
+
+The [metric text of prometheus](./telemetry/metrics.txt),which get
+from `http:{instanceHost}:5801/hazelcast/rest/instance/metrics`.
+
+The [metric text of openMetrics](./telemetry/openmetrics.txt),which get
+from `http:{instanceHost}:5801/hazelcast/rest/instance/openmetrics`.
```

Comment:
The http path looks not right, it missed `//`. Could you create a PR to correct it? @corgy-w

Reply:
get